docs/how-to/build-policies/rebac/overview.mdx

```diff
@@ -3,124 +3,106 @@ sidebar_position: 1
 
-Alice and Bob are working on a project report in Google Docs. Alice creates the main document, and Bob adds supplementary
-material as an appendix in a separate document. Alice then links Bob's appendix to the main report. Now, anyone with
-permission to edit Alice's main report should also automatically gain permission to edit Bob's appendix because of the
-linkage (or relationship) between the two documents.
-
-Check out this [Google Drive example](/modeling/google-drive) being modeled with Permit.
-
-## Basic definitions
-
-### Role Assignment
-
-Role Assignment refers to the act of designating specific roles to individual users or entities, effectively granting them
-the permissions associated with those roles. Roles represent a collection of permissions. By assigning a user a particular role,
-you're essentially giving them access to all the permissions bundled within that role.
-This mechanism simplifies access management, especially in large systems or organizations, as administrators only need to
-manage roles rather than each specific permission for every user.
-
-#### Example
-
-In a company, new employees in the marketing department might be assigned the "Marketing" role, which provides access to
-relevant databases, software, and internal websites without having to configure each access permission individually.
-
-### Resource Role
-
-A Resource Role is a specialized role that's specifically tied to a particular resource.
-This means that the role, and the permissions it carries, is relevant only in the context of that specific resource.
-By creating roles based on resources, organizations can have fine-grained control over who can access and modify specific items.
+### Role Derivation
 
-#### Example
+ReBAC allows us to derive authorization policies based on existing application-level relationships. Put in the simplest way, it allows us to create a policy like this:
 
-In a cloud storage solution like Google Drive, a "Document Editor" role for a specific document might grant permissions to edit
-and comment on that document. However, having the "Document Editor" role for one document doesn't mean the user can edit all
-documents—only the specific one they're assigned to.
+```
+A `user` who is assigned the role of an `Owner` on a folder will also get the `Owner` role on every `file` within that `folder`.
+```
 
-### Role Derivation
+Creating policies based on relationships rather than roles or attributes saves us from having to create authorization policies on a per-instance basis.
 
-Role Derivation is the process or mechanism by which new roles are formulated using existing roles as a foundation.
-This can be useful in hierarchical systems where higher-level roles encompass the permissions of several lower-level roles.
-Role derivation helps in creating roles that are a combination or subset of other roles, providing flexibility and ensuring
-that the principle of least privilege is maintained.
+### Resource Roles
```

Comment:
I would start with the resource roles, as the motivation for role derivation is more explainable after (in RBAC you can also use role inheritance to get the equivalent of role derivation, and it could create some confusion.